src/camera-controls.js

```diff
@@ -694,6 +707,17 @@ export default class CameraControls extends EventDispatcher {
 
 	}
 
+	setBoundary( box3 ) {
+
+		this._boundary.copy( box3 || new THREE.Box3(
```

Comment:
How about this one?
Added default property for the `setBoundary` method

Reply:
Looks good!